tools/mitre/mitredb.py

```diff
@@ -56,12 +195,28 @@ def parse_json(pathfile, session, database):
         metadata = Metadata()
         with open(pathfile) as json_file:
             datajson = json.load(json_file)
-            metadata.version = datajson['spec_version']
-            for data_object in datajson['objects']:
-                if data_object['type'] == 'identity':
-                    metadata.name = data_object['name']
-                elif data_object['type'] == 'marking-definition':
-                    metadata.description = data_object['definition']['statement']
+            metadata.version = datajson[const.VERSION_j]
+            for data_object in datajson[const.OBJECT_j]:
+                if data_object[const.TYPE_j] == const.IDENTITY_j:
+                    metadata.name = data_object[const.NAME_j]
+                elif data_object[const.TYPE_j] == const.MARKING_DEFINITION_j:
+                    metadata.description = data_object[const.DEFINITION_j][const.STATEMENT_j]
+                elif data_object[const.TYPE_j] == const.INTRUSION_SET_j:
+                    groups = parse_table_(Groups, data_object)
+                    session.add(groups)
+                    session.commit()
+                elif data_object[const.TYPE_j] == const.COURSE_OF_ACTION_j:
+                    mitigations = parse_table_(Mitigations, data_object)
+                    session.add(mitigations)
+                    session.commit()
+                elif data_object[const.TYPE_j] == const.MALWARE_j or \
+                        data_object[const.TYPE_j] == const.TOOL_j:
+                    software = parse_table_(Software, data_object)
+                    session.add(software)
+                    session.commit()
+                elif data_object[const.TYPE_j] == const.RELATIONSHIP_j:
```

Comment:
I woud complete the relationships in a second iteration to ensure the DB has already all the data of the other tables.